Neos.Flow/Classes/Core/LockManager.php

```diff
@@ -54,6 +54,7 @@ public function __construct()
         $lockPath = $this->getLockPath();
         $this->lockPathAndFilename = $lockPath . md5(FLOW_PATH_ROOT) . '_Flow.lock';
         $this->lockFlagPathAndFilename = $lockPath . md5(FLOW_PATH_ROOT) . '_FlowIsLocked';
+        $this->lockHoldingPage = defined('FLOW_LOCKHOLDINGPAGE') ? FLOW_PATH_ROOT . FLOW_LOCKHOLDINGPAGE : FLOW_PATH_FLOW . 'Resources/Private/Core/LockHoldingStackPage.html';
```

Comment:
Note: Maybe make that relative to `FLOW_PATH_PACKAGES` instead? Guess there's no much use in putting the page directly into the flow root.